spec/services/metrics/metrics_processor_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative '../../support/metrics_specs_helper'
+
+RSpec.describe Metrics::MetricsProcessor do
```

Comment:
This metrics processor works with multiple specific processors at the same time, maybe we can add tests for that case.